language/stdlib/transaction_scripts/peer_to_peer_with_metadata.move

```diff
@@ -5,6 +5,7 @@ use 0x1::LibraAccount;
 /// associated metadata `metadata` and (optional) `signature` on the metadata, amount, and
 /// sender address. The `metadata` and `signature` parameters are only required if
 /// `amount` >= 1_000_000 micro LBR and the sender and recipient of the funds are two distinct VASPs.
+/// Standardized `metadata` LCS format can be found in `libra_types::transaction::metadata::MetadataType`.
```

Comment:
@n4ss Thanks! Ideally, now, you would run `(cd language/stdlib && cargo run --release)` to upgrade the doc in generated files. (This is going to take a while to compile, sorry :-|)

Reply:
Doing that now!